### DISCUSSION TITLE
后端地址问题

### DISCUSSION THREAD

**Discussion by steveji2021** [MetaCubeX/metacubexd#1030] [Q&A] [answered]
有没有大佬知道填写的后端地址是（网页服务器能访问的地址）还是（客户端浏览器直接访问的地址）？谢谢了

**Comment by kunish** [comment]
没太理解，【网页服务器能访问的地址】和【客户端浏览器直接访问的地址】这两个什么区别？

**Reply by steveji2021** [reply]
就是说，通过单独的web服务器（如caddy）挂载metacubexd网页文件，客户端访问时是【直接在客户端与mihomo之间建立连接，metacubexd服务端只提供静态网页】还是【服务端与mihomo后端建立连接获取信息，然后将动态的网页发给客户端】？

**Comment by Plutonium141** [comment, marked as answer]
应该是（客户端浏览器直接访问的地址）

**Reply by steveji2021** [reply]
好，感谢
